handler/nats_config.go

```diff
@@ -2,15 +2,27 @@ package handler
 
 import (
 	"os"
+	"time"
 
 	"github.com/openfaas/nats-queue-worker/nats"
 )
 
 type NatsConfig interface {
 	GetClientID() string
+	GetMaxReconnect() int
+	GetDelayBetweenReconnect() time.Duration
 }
 
 type DefaultNatsConfig struct {
+	maxReconnect          int
+	delayBetweenReconnect time.Duration
+}
+
+func NewDefaultNatsConfig() DefaultNatsConfig {
+	return DefaultNatsConfig{
+		maxReconnect: 5,
```

Comment:
Question is if we want to have it here? Because we can construct own NatsConfig in gateway and queue-worker and there implement logic for environmental variables

Reply:
Perhaps `NewDefaultNatsConfig` needs to accept these two variables?

Reply:
Done